README.md

```diff
@@ -626,6 +626,171 @@ After this operation, all the class variables are frozen, in order to prevent ac
 
 **This is not necessary, when Lotus::View is used within a Lotus application.**
 
+### Security
+
+The output of views and presenters is always **autoescaped**.
+
+**ATTENTION:** In order to prevent XSS attacks, please read the instructions below.
+Because Lotus::View supports a lot of template engines, the escape happens at the level of the view.
+Most of the times everything happens automatically, but there are still some corner cases that need your manual intervention.
+
+#### View autoescape
+
+```ruby
+require 'lotus/view'
+
+User = Struct.new(:name)
+
+module Users
+  class Show
+    include Lotus::View
+
+    def user_name
+      user.name
+    end
+  end
+end
+
+# ERB template
+# <div id="user_name"><%= user_name %></div>
+
+user = User.new("<script>alert('xss')</script>")
+
+# THIS IS USEFUL FOR UNIT TESTING:
+template = Lotus::View::Template.new('users/show.html.erb')
+view     = Users::Show.new(template, user: user)
+view.user_name # => "&lt;script&gt;alert(&apos;xss&apos;)&lt;&#x2F;script&gt;"
+
+# THIS IS THE RENDERING OUTPUT:
+Users::Show.render(format: :html, user: user)
+# => <div id="user_name">&lt;script&gt;alert(&apos;xss&apos;)&lt;&#x2F;script&gt;</div>
+```
+
+#### Presenter autoescape
+
+```ruby
+require 'lotus/view'
+
+User = Struct.new(:name)
+
+class UserPresenter
+  include Lotus::Presenter
+end
+
+user      = User.new("<script>alert('xss')</script>")
+presenter = UserPresenter.new(user)
+
+presenter.name # => "&lt;script&gt;alert(&apos;xss&apos;)&lt;&#x2F;script&gt;"
+```
+
+#### Escape entire objects
+
+We have seen that concrete methods are in views are automatically escaped.
+This is great, but tedious if you need to print a lot of informations from a given object.
```

Comment:
Should this be:
'We have seen that concrete methods in views are automatically escaped. This is great, but tedious if you need to print a lot of information from a given object.'